types/readdir-glob/index.d.ts

```diff
@@ -0,0 +1,131 @@
+// Type definitions for readdir-glob 1.1
+// Project: https://github.com/Yqnn/node-readdir-glob
+// Definitions by: Dolan Miu <https://github.com/dolanmiu>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="node" />
+
+import * as fs from 'fs';
+import { EventEmitter } from 'events';
+
+export as namespace readdirGlob;
+
+declare function G(root: string, options: G.Options): G.ReaddirGlob;
```

Comment:
Please stick to export JS symbols, so `readdirGlob` looks like more proper name here:
```js
readdirGlob
[Function: readdirGlob] {
  ReaddirGlob: [class ReaddirGlob extends EventEmitter]
```

Reply:
Ok, commited the suggestions.

Can this be resolved?